internal/infrastructure/db/migrations/20230508160305_add_token_id_integrations.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+SET search_path = device_definitions_api, public;
+ALTER TABLE integrations ADD token_id numeric(78,0) null;
```

Comment:
Minor points:

* I wouldn't mind this being an `int`. I don't think there will be too many.
* For some reason I find `ADD COLUMN` comforting to see, even though the `COLUMN` is optional.
* Contradicting myself immediately: I would drop the last `null`.

Reply:
I thought the field was meant to be nullable @elffjs

Reply:
It should be nullable, but that's the default. It's not wrong!